### ISSUE TITLE
Bug: proposeToRemoveGC fails on testrpc, not on kovan

### ISSUE THREAD

**Issue by dkent600** [daostack/daostack#143]
Identical code in Alchemy succeeds with Kovan but fails with testrpc when calling `GlobalConstraintRegistrar.proposeToRemoveGC`.

Failure case triggers "invalid opcode" exception.

**Comment by leviadam** [comment]
What function is this?
There is no such function.

**Comment by dkent600** [comment]
Oops, `proposeToRemoveGC`.  Corrected.

**Comment by leviadam** [comment]
Will look into it.

**Comment by leviadam** [comment]
Do you have all things fully updated?
Maybe go `yarn`?

**Comment by dkent600** [comment]
This is with all the latest.  I haven't tried reproducing the error in the Arc automated test code (hence yarn is irrelevant), rather I have observed it in Alchemy.

**Comment by dkent600** [comment]
@leviadam I just confirmed that this error happens in automated test code too.

**Comment by leviadam** [comment]
Hi @dkent600, can you elaborate how you checked it? All my tests are running.

**Comment by dkent600** [comment]
Using branch #98, run this test:

```
    it("proposeToRemoveGlobalConstraint javascript wrapper should not crash", async function() {
    const organization = await helpers.forgeOrganization();

    let tokenCapGC = await organization.scheme('TokenCapGC');

    let globalConstraintRegistrar = await organization.scheme('GlobalConstraintRegistrar');

    await globalConstraintRegistrar.proposeToRemoveGlobalConstraint({
      avatar: organization.avatar.address,
      globalConstraint: tokenCapGC.address
    });

    // const proposalId = getValueFromLogs(tx, '_proposalId');

    // console.log(`****** proposal ID ${proposalId} ******`);
  });
```

**Comment by dkent600** [comment]
This is still happening for me

**Comment by orenyodfat** [comment]
This issue is fixed at pr #193 
I guess that the root cause of that was missing of update of the removeParam on adding a new global constraint . [here](https://github.com/daostack/daostack/pull/193/files#diff-e01ebd27b4b9fc22923deecf89f4fee9R163) .
